core-bundle/src/InsertTag/InsertTag.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\InsertTag;
+
+abstract class InsertTag
```

Comment:
Should we make this `@internal`? The two implementers are already final and I don't think anyone is supposed to use this class, but this way we could change things up in the future without having to think about BC.

(Same reasoning applies for `InsertTagParameters`.)

Reply:
Done in #6195